editor/filesystem_dock.h

```diff
@@ -171,6 +171,7 @@ class FileSystemDock : public VBoxContainer {
 	MenuButton *file_list_button_sort = nullptr;
 
 	String searched_string;
```

Comment:
Seems like this is not really used anymore. There are only checks whether the string is not empty, which can be replaced by `searched_tokens.is_empty()`.